src/backend/TrafficCourts/Workflow.Service/Models/TcoDisputeTicket.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Text;
+
+namespace TrafficCourts.Workflow.Service.Models
+{
+    public class TcoDisputeTicket
+    {
+        public string CitizenName { get; set; }
+        public DateTime TicketIssuanceDate { get; set; }
+        public string TicketFileNumber { get; set; }
+        public string IssuingOrganization { get; set; }
+        public string IssuingLocation { get; set; }
+        public string DriversLicense { get; set; }
+        public List<TicketDetails> TicketDetails { get; set; }
```

Comment:
I'm seeing a nullable warning on all the above properties.
"A non-null property must be populated when leaving the constructor, or marked as nullable"